frontend/src/libs/components/input/input.tsx

```diff
@@ -42,19 +48,36 @@ const Input = <T extends FieldValues>({
     hasError && styles.error,
   ];
 
+  const toggleShowPassword = useCallback(
+    (event: React.MouseEvent<HTMLElement>): void => {
+      event.preventDefault();
+      setIsShowPassword(!isShowPassword);
+    },
+    [isShowPassword],
+  );
+
   return (
     <label className={styles.inputComponentWrapper}>
       {hasLabel && <span className={styles.label}>{label}</span>}
       <span className={styles.inputWrapper}>
         <input
           {...field}
-          type={type}
+          type={isShowPassword ? 'text' : type}
           placeholder={placeholder}
           className={getValidClassNames(...inputStyles)}
           disabled={isDisabled}
         />
         {type === 'password' && (
-          <span className={styles.passwordEye}>&#128065;</span>
+          <button
+            className={getValidClassNames(
+              styles.passwordEye,
+              isShowPassword && styles.passwordEyeLight,
+            )}
+            onClick={toggleShowPassword}
+            tabIndex={-1}
```

Comment:
do we need tabIndex with button here?

Reply:
If remove it when tabing through form eye symbol will be in focus,  It seems for me as not predictable behavior